apps/meteor/ee/app/livechat-enterprise/server/hooks/onSaveVisitorInfo.ts

```diff
@@ -11,7 +11,7 @@ const updateSLA = async (room: IOmnichannelRoom, user: Required<Pick<IUser, '_id
 		return removeRoomSLA(room._id, user);
 	}
 
-	const sla = await OmnichannelServiceLevelAgreements.findOneById(slaId, { projection: { _id: 1, dueTimeInMinutes: 1 } });
+	const sla = await OmnichannelServiceLevelAgreements.findOneById(slaId);
```

Comment:
Does it need the full object? AFAIK, the only things we care of slas are the name and dueTime 👀

Reply:
I mean, what's left then :trollface: ? just the description field :sweat_smile: